# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/x86/apmd/hotfixes.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./apmsleep.c.orig	2001-05-11 22:30:08.000000000 +0200
+++ ./apmsleep.c	2002-09-15 17:38:51.000000000 +0200
@@ -42,11 +42,15 @@
  *
  */
 
+typedef unsigned char u8;
+typedef unsigned short u16;
+typedef unsigned int u32;
+
 #include <stdio.h>
 #include <linux/version.h>
 
 #if LINUX_VERSION_CODE > KERNEL_VERSION(2,2,0)
-#include <asm/spinlock.h>
+// #include <asm/spinlock.h>
 #endif
 
 #include <linux/mc146818rtc.h>
